client/hack/update-generated-code.sh

```diff
@@ -26,7 +26,7 @@ SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
 ${GOPATH}/src/k8s.io/code-generator/generate-groups.sh "deepcopy,client,informer,lister" \
   github.com/kubernetes-csi/external-snapshotter/client/v3 github.com/kubernetes-csi/external-snapshotter/client/v3/apis \
-  volumesnapshot:v1beta1 \
+  volumesnapshot:v1 \
```

Comment:
Do we want to keep generated client code for both v1beta and v1?

Reply:
Yes.  I expect the existing generated client code for v1beta to stay, but we will only be changing generated client code for v1.  Therefore this code generating script should only handle v1.  I'll add a note there.